build-tests/install-test-workspace/build.js

```diff
@@ -160,12 +160,13 @@ let shrinkwrapUpdatedNotice = false;
 
 if (pnpmLockBeforeContent !== pnpmLockAfterContent) {
   if (productionMode) {
-    console.error('The shrinkwrap file is not up to date:');
-    console.error('  Git copy:     ' + pnpmLockBeforePath);
-    console.error('  Current copy: ' + pnpmLockAfterPath);
-    console.error('\nPlease commit the updated copy to Git\n');
-    process.exitCode = 1;
-    return;
+    // TODO: Re-enable when issue with lockfile diffing is resolved
+    // console.error('The shrinkwrap file is not up to date:');
+    // console.error('  Git copy:     ' + pnpmLockBeforePath);
+    // console.error('  Current copy: ' + pnpmLockAfterPath);
+    // console.error('\nPlease commit the updated copy to Git\n');
+    // process.exitCode = 1;
+    // return;
```

Comment:
@octogonz - You good with this? This issue has gotten fairly disruptive.

Reply:
I spoke to him offline, his suggestion was to remove the diff check:
> If you get stuck and can't figure it out, I'd say just disable the --production diffing